lib/Dialect/Secret/IR/SecretOps.cpp

```diff
@@ -337,6 +337,38 @@ GenericOp GenericOp::removeYieldedValues(ValueRange yieldedValuesToRemove,
   return cloneWithNewTypes(*this, newResultTypes, rewriter);
 }
 
+GenericOp GenericOp::removeYieldedValues(ArrayRef<int> yieldedIndicesToRemove,
+                                         PatternRewriter &rewriter,
+                                         SmallVector<Value> &remainingResults) {
+  YieldOp yieldOp = getYieldOp();
+  for ([[maybe_unused]] int index : yieldedIndicesToRemove) {
+    assert(0 <= index && index < yieldOp.getNumOperands() &&
+           "Cannot remove an index that is out of range");
+  }
+
+  SmallVector<int, 4> indicesToErase;
```

Comment:
nit: remove this since i think it's unused

Reply:
Fixed!